experiments/long_runs/snowy_land_pmodel.jl

```diff
@@ -120,13 +121,22 @@ function setup_model(
         forcing,
         toml_dict,
         Δt;
-        prognostic_land_components = (:canopy, :snow, :soil, :soilco2),
+        prognostic_land_components,
         α_snow,
         scf,
     )
 
     # Construct the land model with all default components except for snow
-    land = LandModel{FT}(forcing, LAI, toml_dict, domain, Δt; snow, canopy)
+    land = LandModel{FT}(
```

Comment:
similar changes for LandModel run regionally. Do we need to update any of the benchmark scripts, calibration script also? 

I think it is OK for those all to be the "default" without soilco2, if that amounts to less changes, but keeping the current behavior in snowy_land and snowy land_pmodel is good for now

Reply:
For calibration though I think we can use no soilco2 - it doesnt affect what we are calibrating right now, and slows it down!